config.go

```diff
@@ -32,6 +37,8 @@ type Config struct {
 	// Consul. This requires Consul to be configured to serve HTTPS.
 	SSL bool `mapstructure:"ssl"`
 
+	HttpAuth HttpAuth `mapstructure:"HttpAuth"`
```

Comment:
The `mapstructure` tag maps the value to the key in a config file, which should follow the snake-chaining (lowercase with underscores) convention.